meson.build

```diff
@@ -402,6 +402,7 @@ possible_link_flags = [
         '-Wl,-z,relro',
         '-Wl,-z,now',
         '-fstack-protector',
+        '-lgnutls'
```

Comment:
shouldn't this get automatically added via the dependency below?

Reply:
@yuwata ?

Reply:
It will - that's what dependencies does. You can remove this line.

Reply:
Bi this does not fix

Reply:
it wasn't about fixing the build, it's an independent issues - library linking flags are added by Meson using pkg-config, they shouldn't be added manually